data/announcement.json

```
@@ -1,5 +1,5 @@
 {
     "enabled": true,
-    "text": "XMPP Summit 28 - 29th & 30th Jan 2026",
+    "text": "XMPP Summit 28  ||  29th - 30th Jan 2026",
```

Double-barrel seems arbitrary for separating title and date. How about this:

```suggestion
    "text": "XMPP Summit 28 - Join us in Brussels, 29th - 30th Jan 2026",
```